src/cli/bin.ts

```diff
@@ -122,6 +129,25 @@ yargs
         default: false,
         desc: "Whether stacktraces are included in the error report file.",
       });
+      args.option("on-error", {
+        type: "string",
+        demandOption: false,
+        choices: [
+          "keep-original",
+          "keep-last-successful-version",
+          "keep-going",
+        ],
+        default: "keep-original",
+        desc: `The behavior when an error occurs during the migration of an item. This has an effect only when migrating through several versions in one go.
```

Comment:
FYI, this what it gives in a terminal:
![image](https://user-images.githubusercontent.com/15254842/214702481-a174f4ff-ea74-4cde-8292-527c4c2c5ad7.png)